### PR TITLE
Graph run tests

### LINKED ISSUE #111: Test that graph functions don't fail

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #111 

This PR adds tests for:
* The funding waterfall chart
* The decile chart
* The poverty chart
* The household waterfall chart
* The budget line chart
* The MTR chart

On the population side, the tests run for a set of example reforms:
* Empty reform
* Abolition of the personal allowance
* Raising of the basic rate

On the household side, the tests run for each of the above reforms, for each of a set of example situations:
* Single adult, age 18

I had to make the Flask server code a Python package in order to run the tests with pytest (couldn't import from multiple levels up).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from pathlib import Path
+
+setup(
+    name="PolicyEngine-UK",
+    version="0.1.0",
+    author="PolicyEngine",
+    license="http://www.fsf.org/licensing/licenses/agpl-3.0.html",
+    url="https://github.com/ubicenter/policyengine-uk",
+    install_requires=[
+        # "OpenFisca-UK @ git+https://github.com/pslmodels/openfisca-uk",
```
Comment (MaxGhenis) [inline setup.py:13]: remove?

File: tests/server/situations/test_situation_charts.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+)
+
+reform_examples = (
+    empty_reform,
+    abolish_personal_allowance,
+    raise_basic_rate,
+)
+
+
+@pytest.mark.parametrize(
```
Comment (MaxGhenis) [inline tests/server/situations/test_situation_charts.py:35]: This is cool - could you add a comment about what it's doing (every combination of example households and reforms)?

**Review by MaxGhenis — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (nikhilwoodruff) [reply setup.py:13]: Could do - wasn't really sure, technically it does require if you were to install the package by itself, it's just that GCP needs the repo clone hack

File: tests/server/situations/test_situation_charts.py (hunk 2 above)
Comment (nikhilwoodruff) [reply tests/server/situations/test_situation_charts.py:35]: Added a comment - thanks!

File: setup.py (hunk 1 above)
Comment (MaxGhenis) [reply setup.py:13]: OK that's fine just add a comment then about why it's commented

**Pull request merged**